packages/jsii-rosetta/lib/tablets/tablets.ts

```diff
@@ -1,5 +1,6 @@
 import * as fs from 'fs-extra';
 import * as path from 'node:path';
+import * as zlib from 'node:zlib';
```

Comment:
reminder that this import will need to be updated when node protocol is reverted: #3651